pkg/resource/v1/worker.go

```diff
@@ -9,6 +9,8 @@ type Worker struct {
 
 	MachineID string
 
+	Unschedulable bool
```

Comment:
What do yo think of `SchedulingPaused` name instead?

Reply:
I'm fine with it, renamed in https://github.com/cirruslabs/orchard/pull/73/commits/0a29be3254d4d91cb18fdc5b9aca0eef48928517.